### ISSUE TITLE
Logging configuration

### ISSUE THREAD

**Issue by TheCodeKing** [kamsar/Unicorn#176]
We use Octopus to deploy unicorn via Powershell and it takes a very long time. I think part of the issue is the verbose logs as it's 1000s of lines. I can't find any documentation about how to reduce this and only show warnings and errors in output. Does this ability exist? We are using Unicorn 3.3.0-RC3. Thanks.

**Comment by kamsar** [comment]
You have thousands of changes on every deploy? Are you wiping environments up from scratch or something? :)

The automated tool method does not currently support log levels. You could implement something similar to the log leveling in: https://github.com/kamsar/Unicorn/blob/master/src/Unicorn/Logging/WebConsoleLogger.cs

Then implement it in the automated tool console:
https://github.com/kamsar/Unicorn/blob/master/src/Unicorn/ControlPanel/UnicornStringConsole.cs

And hook it to a query param to set the verbosity, in https://github.com/kamsar/Unicorn/blob/master/src/Unicorn/ControlPanel/Responses/WebConsoleResponse.cs

**Comment by TheCodeKing** [comment]
OK thanks, I may look at this as logging is too verbose, it's all Debug messages etc. Yes we build from scratch every time and follow Continuous Delivery principles. Once in production we will switch to deltas.

I did find some code already that checks QueryString for log. I modified powershell to send log=warning and it seems to have reduced output, although strangely still getting some debug messages.
